aws/resource_aws_resourcegroups_group_test.go

```diff
@@ -75,7 +63,52 @@ func TestAccAWSResourceGroup_basic(t *testing.T) {
 	})
 }
 
-func testAccCheckAWSResourceGroupExists(n string) resource.TestCheckFunc {
+func TestAccAWSResourceGroup_tags(t *testing.T) {
+	var v resourcegroups.Group
+	resourceName := "aws_resourcegroups_group.test"
+	n := fmt.Sprintf("test-group-%d", acctest.RandInt())
+	desc1 := "Hello World"
```

Comment:
Nit: This acceptance testing does not need to verify anything with this attribute, so it should omitted or hardcoded in the test configurations.